docs_devel/docs/12.HowToMakeMTConnector.md

```diff
@@ -0,0 +1,112 @@
+# How to make Machine Translation connector plugin
+
+## What is MT connector
+
+OmegaT can be extended with plugin. One of the popular categories of plugin is a Machine Translation (MT) connector.
+There are many translation web services, thanks to the innovation of AI technologies.
+OmegaT bundles some MT connectors that support popular services, such as Google, DeepL and IBM Watson.
+There are also genuine and third party plugins for services;
+
+- [Azure translate](https://github.com/omegat-org/azure-translate-plugin/releases)
+- [Moses](https://github.com/omegat-org/moses-plugin/releases)
+- [Autshumato MT](https://sourceforge.net/projects/autshumatoite/files/OmegaT-plugins-AutshumatoMT/)
+- [Mirai translator](https://codeberg.org/miurahr/omegat-mirai/releases)
+- [NICT TexTra](https://codeberg.org/miurahr/omegat-textra-plugin/releases)
+- [Tencent translation](https://github.com/yoyicue/omegat-tencent-plugin)
+
+## Getting started the project
```

Comment:
The review comments  tell us a suggestion that developers may  want to know a way to write unit tests, to run integration test, and a way to run implemented features without learning things which he should know.
We can explain about WireMock and give a reference URL when developers want to learn it for unit tests.
We can also explain a parser test example which is mandatory for unit test.

Reply:
> The review comments tell us a suggestion that developers may want to know a way to write unit tests, to run integration test, and a way to run implemented features without learning things which he should know.

If you tag me for review and you don't tell me explicitly what I'm supposed to do and what I'm supposed to expect, I'm not sure my review will be very relevant.

Also, there is no such thing that I "should know". There are things that you know, things that I know, and things that I can learn, but I can't guess what I have to learn if it is not explicitly stated, and then it depends on my general availability.

Reply:
It is challenging for me to express a fact and ideas correctly. I have an idea to provide a way to test a MT feature of OmegaT without knowing actual MT services, and want to write a document about it.
I will update a topic branch with a feature and description how to use it. This will allow developers to see a behavior without learning external services/products other than OmegaT.

Reply:
Thank you.

Reply:
pls. see an update.